code/datums/quirks/good.dm

```diff
@@ -1,6 +1,56 @@
 //predominantly positive traits
 //this file is named weirdly so that positive traits are listed above negative ones
 
+/datum/quirk/resistant
+	name = "Damage resistant"
+	desc = "You take 5% less damage from all sources"
+	value = 10
+	gain_text = "<span class='notice'>You feel a bit more resistant to damage than others.</span>"
+	medical_record_text = "This patient is observed to be able to withstand a little bit more damage before going unconcious."
+	hardcore_value = -8
+
+/datum/quirk/resistant/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.physiology.red_mod -= 0.05
+	H.physiology.white_mod -= 0.05
+	H.physiology.black_mod -= 0.05
+	H.physiology.pale_mod -= 0.05
+
+/datum/quirk/skilled
+	name = "Prepared for anything"
+	desc = "You gain an extra 5 permanent attribute points to every aspect by training."
+	value = 20
+	gain_text = "<span class='notice'>You look around to see people less prepared than you.</span>"
+	medical_record_text = "This patient is observed to have unnatural abs."
+	hardcore_value = -40
+
+/datum/quirk/skilled/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.adjust_attribute_buff(FORTITUDE_ATTRIBUTE, 5)
+	H.adjust_attribute_buff(PRUDENCE_ATTRIBUTE, 5)
```

Comment:
I dont know if we should have a perk that is just one of each trait... I mean a small health bonus or sanity bonus might be better since that only gives so much additional survivability. Also isnt the all around resist bonus a bit much? I mean 5% is very small but it feels like it would just become the norm. Also like the pineapple on pizza some aspects should be exclusive.

Reply:
yeah, probably a good idea to remove the good quirks as the note says. gonna want to hear further feedback from other on that front tho